src/components/Profile/TitleSection/TitleSection.styled.js

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+
+export const TitleProfileSection = styled.div`
+  display: flex;
+  align-items: center;
+  margin-bottom: 15px;
+  padding: 15px 0;
+  border-bottom: 1px solid var(--color-border-secondary);
+`;
+
+export const Wrapper = styled.div`
+  width: 100%;
+  margin-left: 10px;
+
+  > div {
+    display: flex;
+    justify-content: space-between;
+
+    p.name {
```

Comment:
Why is it p.name and not just " .name "?

Reply:
fix